shell/platform/common/accessibility_bridge.cc

```diff
@@ -177,6 +199,59 @@ void AccessibilityBridge::OnAtomicUpdateFinished(
   }
 }
 
+std::optional<ui::AXTreeUpdate>
+AccessibilityBridge::CreateRemoveReparentedNodesUpdate() {
+  std::unordered_map<int32_t, ui::AXNodeData> updates;
+
+  for (auto node_update : pending_semantics_node_updates_) {
+    for (int32_t child_id : node_update.second.children_in_traversal_order) {
+      // Skip nodes that don't exist or have a parent in the current tree.
+      ui::AXNode* child = tree_.GetFromId(child_id);
+      if (!child || !child->parent()) {
+        continue;
+      }
+
+      // Skip nodes whose parents are unchanged.
+      if (child->parent()->id() == node_update.second.id) {
+        continue;
+      }
+
+      // This pending update moves the current child node.
+      // That new child must have a corresponding pending update.
+      assert(pending_semantics_node_updates_.find(child_id) !=
+             pending_semantics_node_updates_.end());
+
+      // Create an update to remove the child from its previous parent.
+      int32_t parent_id = child->parent()->id();
+      if (updates.find(parent_id) == updates.end()) {
+        updates[parent_id] = tree_.GetFromId(parent_id)->data();
+      }
+
+      ui::AXNodeData* parent = &updates[parent_id];
+      auto iter = std::find(parent->child_ids.begin(), parent->child_ids.end(),
+                            child_id);
+
+      assert(iter != parent->child_ids.end());
+      parent->child_ids.erase(iter);
+    }
+  }
+
+  if (updates.empty()) {
+    return std::nullopt;
+  }
+
+  ui::AXTreeUpdate update = {
```

Comment:
Shorter and sweeter - just use initializer syntax:
```suggestion
  ui::AXTreeUpdate update{
```

Reply:
Fixed!